### ISSUE TITLE
Using {{content}}, {{property}}

### ISSUE THREAD

**Issue by eueung** [gnab/remark#232]
just tried http://gnab.github.com/editorjs with the latest remark build.

It seems that

```
name: default
layout: true
task: &nbsp;

.task[{{task}}]

---
task: [#01 Object properties](http://js-workshop.herokuapp.com/#1/1)

# Objects

```

is no longer working since 0.6.2 onwards. Is there any other method to achieve this in the recent versions?

EM

**Comment by tbfe-de** [comment]
Isn't this the bug I described in #357 and for which I supplied a fix in v0.14?

If so, this issue may be closed now.

**Comment by eueung** [comment]
this is a two-year old issue and the Iatest version I've checked is 0.11 ... from description in #357, it seems you might be right.

So I'll close this.
